persistence-modules/spring-data-jpa-repo-3/pom.xml

```diff
@@ -3,16 +3,17 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Add a space between dependencies and properties

Reply:
done